src/main/resources/messages/help_ru.yml

```diff
@@ -35,11 +35,122 @@ section:
+        detailedDescription: 'Убирает регистрацию указанного игрока из базы данных.'
+        arg1:
+            label: 'игрок'
+            description: 'имя игрока'
+    authme.forcelogin:
+        description: 'Авторизация игрока'
+        detailedDescription: 'Авторизация указанного игрока.'
+        arg1:
+            label: 'игрок'
+            description: 'имя игрока в сети'
+    authme.password:
+        description: 'Изменение пароля игрока'
+        detailedDescription: 'Изменяет пароль указанного игрока.'
+        arg1:
+            label: 'игрок'
+            description: 'имя игрока'
+        arg2:
+            label: 'пароль'
+            description: 'новый пароль'
+    authme.lastlogin:
+        description: 'Последний вход игрока'
+        detailedDescription: 'Показывает дату и время последнего входа указанного игрока.'
+        arg1:
+            label: 'игрок'
+            description: 'имя игрока'
+    authme.accounts:
+        description: 'Учётные записи игрока'
+        detailedDescription: 'Отображает все учётные записи указанного игрока по его имени и IP-адресу.'
+        arg1:
+            label: 'игрок'
+            description: 'имя игрока или IP-адрес'
+    authme.email:
+        description: 'Электронная почта игрока'
+        detailedDescription: 'Отображает адрес электронной почты указанного игрока.'
+        arg1:
+            label: 'игрок'
+            description: 'имя игрока'
+    authme.setemail:
+        description: 'Смена эл. почты игрока'
+        detailedDescription: 'Изменяет адрес электронной почты указанного игрока.'
+        arg1:
+            label: 'игрок'
+            description: 'имя игрока'
+        arg2:
+            label: 'эл. почта'
+            description: 'электронная почта игрока'
+    authme.getip:
+        description: 'IP-адрес игрока'
+        detailedDescription: 'Показывает IP-адрес указанного игрока в сети.'
+        arg1:
+            label: 'игрок'
+            description: 'имя игрока'
+    authme.spawn:
+        description: 'Перемещение на точку возрождения'
+        detailedDescription: 'Перемещает на точку возрождения.'
+    authme.setspawn:
+        description: 'Перемещение точки возрождения'
+        detailedDescription: 'Перемещает точку возрождения игроков на ваше текущее местоположение.'
+    authme.firstspawn:
+        description: 'Перемещение на начальную точку появления'
+        detailedDescription: 'Перемещает на начальную точку появления.'
+    authme.setfirstspawn:
+        description: 'Перемещение начальной точки появления'
+        detailedDescription: 'Перемещает начальную точку появления игроков на ваше текущее местоположение.'
+    authme.purge:
+        description: 'Удаление старых данных'
+        detailedDescription: 'Удаляет старые данные старше указанного количества дней.'
+        arg1:
+            label: 'дни'
+            description: 'количество дней'
+        arg2:
+            label: 'all'
```

Comment:
Missing translation for the label

Reply:
![image](https://user-images.githubusercontent.com/17567951/33444325-048f0560-d60b-11e7-8819-cdfef7614b58.png)
«all» is like «Mansion» of the «/locate Mansion».